src/cascadia/TerminalControl/XamlUiaTextRange.cpp

```diff
@@ -189,9 +189,9 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
     winrt::hstring XamlUiaTextRange::GetText(int32_t maxLength) const
     {
-        BSTR returnVal;
-        THROW_IF_FAILED(_uiaProvider->GetText(maxLength, &returnVal));
-        return winrt::to_hstring(returnVal);
+        wil::unique_bstr returnVal;
+        THROW_IF_FAILED(_uiaProvider->GetText(maxLength, returnVal.put()));
+        return winrt::to_hstring(returnVal.get());

```

Comment:
We could choose to use `return winrt::hstring{ returnVal.get(), SysStringLen(returnVal.get()) }` here to avoid the need for the `wcslen` call.